### DISCUSSION TITLE
Customized components and accessibility

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5332] [Accessibility]
Greetings, I'm the USWDS Accessibility Specialist and wanted to reach out for input from the community.

I know it's pretty common for teams to customize components (for example, the accordion, button, etc.) to create a new pattern/interactive element.

My question is: What are your expectations for the accessibility of customized components? Do you expect to re-check these components for proper accessibility? Or do you expect that most (or all) of the accessibility functionality of the original component will apply to the custom version? How do you decide?

Feel free to add examples, feedback, and your thoughts on the topic. I appreciate your input!

**Comment by mgifford** [comment]
I do think that this would likely depend on the person's role and experience with accessibility. For that matter what the customization is. 

My assumption is that many teams will make customizations and not know when the semantics have been altered by their modifications. My sense is that most teams are not confident in how HTML/CSS affects the semantics of a component. The focus is almost entirely on look/feel and mouse driven interactions. 

Would love to hear I am wrong about this, but not sure how people decide.
